src/util.rs

```diff
@@ -119,7 +119,7 @@ fn formats() -> impl Parser<BitFlags<Format>> {
 		let mut formats = BitFlags::empty();
 
 		#[rustfmt::skip]
-		let _ = {
+		{
```

Comment:
This won't work because attributes [only work with *statements* and not expressions](https://github.com/rust-lang/rust/issues/15701) — `let`s are statements; a  block isn't.